src/Objs/_metadataEditingConfig.js

```diff
@@ -25,4 +25,44 @@ export const metadataPropertiesGroup = {
 export const socialCardsPropertiesGroup = {
   title: "Social cards",
   component: "SocialCardsTab",
+  properties: ["tcCreator", "tcDescription", "ogDescription"],
 };
+
+export const socialCardsValidations = [
+  [
+    "tcCreator",
+
+    tcCreator => {
+      if (tcCreator && tcCreator.charAt(0) !== "@") {
+        return {
+          message: "The username should start with @.",
+          severity: "warning",
+        };
+      }
+    },
+  ],
+  [
+    "tcDescription",
+
+    tcDescription => {
+      if (tcDescription && tcDescription.length > 200) {
+        return {
+          message: "The description should not exceed 200 characters.",
```

Comment:
Since the message is also shown outside of the "social cards" extension we should differentiate which description is meant:

```suggestion
          message: "The Twitter description should not exceed 200 characters.",
```

Otherwise it's hard to distinguish the error message:

![validation list](https://user-images.githubusercontent.com/86275/71891901-eb662380-3147-11ea-93bc-86712f04d990.png)